app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(comand: str) -> None:
+    names = comand.split()
```

Comment:
You can unpack the list which you will get from split() method like this:

`variable_a, variable_b, variable_c = "a b c".split()`